### PR TITLE
Contrib Updates Part 1

### PR DESCRIPTION
**Pull request by Renrut5**
First step in suggested changes for contrib files. Goal is to make a number of changes to improve usability based on my experiences with contrib but without changing current functionality.

### CODE REVIEW

File: nautobot_ssot/contrib/utils.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+"""Supporting classes for SSoT contib interfacing with Nautobot."""
```
Comment (Kircheneer) [inline nautobot_ssot/contrib/utils.py:1]:
```suggestion
"""Supporting classes for SSoT contrib interfacing with Nautobot."""
```

File: nautobot_ssot/contrib/utils.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+from enum import Enum
+from dataclasses import dataclass
+
+
+class RelationshipSideEnum(Enum):
+    """This details which side of a custom relationship the model it's defined on is on."""
+
```
Comment (Kircheneer) [inline nautobot_ssot/contrib/utils.py:9]: Can we rename this file to `types.py`?

**Review by Kircheneer — CHANGES_REQUESTED** [review]
I like it! Just two minor changes and a bit of CI and then we're good to go.

**Comment by Renrut5** [comment]
@Kircheneer , got the requested changes made, once the pipeline finishes should be good to merge

**Review by jdrew82 — APPROVED** [review]
LGTM!

**Pull request merged**